### DISCUSSION TITLE
Kitsu sync support

### DISCUSSION THREAD

**Discussion by Atkoset** [xgi/houdoku#138] [Feature Requests]
Adding the Kitsu Anime and Manga tracker to houdoku.

**Comment by Atkoset** [comment]
Hello, could you add Kitsu API too if possible? Kitsu is comparable to Anilist in the way that it also syncs your anime. Where it differs is that it's fully opensource option that a lot of people including me use since it's opensource it shouldn't take too much work to integrate I hope. I have included a link to their API repo below.

https://github.com/hummingbird-me/api-docs
@xgi

**Comment by Atkoset** [comment]
They also have documentation on how to do it.

**Comment by xgi** [comment]
Thanks for the request- I've worked a bit with that API before and it should be possible to add support, but with any integration there's a fair bit of work to map Kitsu's schema with Houdoku's schema. I'll try to get around to this, but I can't prioritize it immediately. I have created a backlog task: #148
